lib/polyfill/media_capabilities.js

```diff
@@ -201,6 +217,34 @@ shaka.polyfill.MediaCapabilities = class {
 
     return res;
   }
+
+  /**
+   * @param {!string} videoCodec
+   * @param {!string} audioCodec
+   * @param {!string} keySystem
+   * @return {!string}
+   * @static
+   */
+  static generateKeySystemCacheKey(videoCodec, audioCodec, keySystem) {
```

Comment:
Please annotate this with `@private` and end the name with an underscore.

Reply:
:+1: